### DISCUSSION TITLE
Stuck on "TLS certificate is being provisioned"

### DISCUSSION THREAD

**Discussion by franciscopower** [community/community#36235] [Other Feature Feedback, Questions, & Ideas]
Hi! 
I'm trying to create my first GitHub pages website, using a custom subdomain, from Google Domains.
I have my apex domain (`itspower.dev`) pointing to a server, **not** related to GitHub, using a DDNS, and I created a CNAME record (`wedding.itspower.dev`) pointing to `franciscopower.github.io`.

However, I have been stuck for hours in step 1 of "_TLS certificate is being provisioned_". Is there something I'm doing wrong?

I tried debugging with the "dig" command, but I found nothing wrong...

```bash
; <<>> DiG 9.16.1-Ubuntu <<>> wedding.itspower.dev
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 40401
;; flags: qr rd ad; QUERY: 1, ANSWER: 5, AUTHORITY: 0, ADDITIONAL: 0
;; WARNING: recursion requested but not available

;; QUESTION SECTION:
;wedding.itspower.dev.          IN      A

;; ANSWER SECTION:
wedding.itspower.dev.   0       IN      CNAME   franciscopower.github.io.
franciscopower.github.io. 0     IN      A       185.199.108.153
franciscopower.github.io. 0     IN      A       185.199.109.153
franciscopower.github.io. 0     IN      A       185.199.110.153
franciscopower.github.io. 0     IN      A       185.199.111.153
```

**Comment by yoannchaudet** [comment]
I cannot lookup any Pages site with domain `wedding.itspower.dev` so I assume you removed it. We have seen an uptick of issues with TLS certificates provisioning last weekend which has been resolved since. You may give it another try. Please note that DNS propagation can affect how long it takes for GitHub to obtain a certificate for your domain (from minutes to hours).

**Reply by franciscopower** [reply]
Yes, I was on a tight schedule to get the website up and I had to find a workaround (a URL redirect, basically). However, I will now try again. Thank you for your reply!

**Reply by franciscopower** [reply]
Just checked and it worked. Thanks!

**Comment by ossanna16** [comment]
Hey there @franciscopower and thank you for asking a good question! I'm glad you found our community 🙂
